stdlib.go

```diff
@@ -52,6 +53,16 @@ func MessageKey(key string) StdlibAdapterOption {
 	return func(a *StdlibAdapter) { a.messageKey = key }
 }
 
+// StdlibRegexp sets the regular expression used to parse stdlib log messages.
+// Nil regexps are ignored and will return options that are no-ops. The default
+// value is StdlibRegexpFull.
+func StdlibRegexp(re *regexp.Regexp) StdlibAdapterOption {
```

Comment:
So there are a few requirements of the regexps that can be successfully passed here. For example, if they intend to parse a date, then they need to include a capture group named `date`. Correct? Those requirements need to be documented, I suppose...

Reply:
done

Reply:
Please, feel free to adjust the comment as needed and commit directly to avoid one more loop.